interface.tf

```diff
@@ -58,6 +59,6 @@ output "api_elb_address" {
 }
 
 output "api_host_addresses" {
-  value = [aws_instance.api[*].private_ip]
+  value = aws_instance.api[*].private_ip
```

Comment:
I'm pretty sure this crops up in other places in the book/code too. Without tweaking, I had instances of arrays inside arrays inside of arrays.